examples/src/main/java/io/kubernetes/client/examples/PagerExample.java

```diff
@@ -0,0 +1,76 @@
+/*
+Copyright 2019 The Kubernetes Authors.
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+    http://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+package io.kubernetes.client.examples;
+
+import io.kubernetes.client.ApiClient;
+import io.kubernetes.client.Configuration;
+import io.kubernetes.client.apis.CoreV1Api;
+import io.kubernetes.client.models.V1Namespace;
+import io.kubernetes.client.models.V1NamespaceList;
+import io.kubernetes.client.pager.Pager;
+import io.kubernetes.client.util.Config;
+import io.kubernetes.client.util.PagerParams;
+import java.io.IOException;
+import java.util.List;
+import java.util.concurrent.TimeUnit;
+
+/**
+ * A simple example of how to use the Java API
+ *
+ * <p>Easiest way to run this: mvn exec:java
+ * -Dexec.mainClass="io.kubernetes.client.examples.PagerExample"
+ *
+ * <p>From inside $REPO_DIR/examples
+ */
+public class PagerExample {
+  public static void main(String[] args) throws IOException {
+
+    ApiClient client = Config.defaultClient();
+    client.getHttpClient().setReadTimeout(60, TimeUnit.SECONDS);
```

Comment:
(might be an unrelated question) what's the point of this?

Reply:
This just prevents timeouts for large bodies/slow connections. I think the default is 10 seconds?

It could be that this is cargo-cult copied from other examples :)